.chronus/changes/fix-header-example-2025-4-29-19-16-10.md

```diff
@@ -0,0 +1,8 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: fix
```

Comment:
```suggestion
changeKind: internal
```

Sorry I pushed to your branch directly. This should be internal as its not really worth showing in the changelog